frontend/src/components/MapView/MapTooltip/PopupContent.tsx

```diff
@@ -168,6 +168,10 @@ const PopupContent = ({
           return true;
         })
         .map(([key, value]) => {
+          // If the data is undefined, null, or an empty string, we don't want to show the key/value pair
```

Comment:
Can you explicit the use-case for this? Note that we already have the reverse case where we hide the `key` using the `do_not_display` prefix

Reply:
This allows us to pass a single string to PopupContent to display rather then always displaying a label: data. That's necessary to render the layer name on it's own line without following with `: {data}`. I'll update the comment to make that clearer — "we don't want to show the key/value pair" doesn't mean much 😅
<img width="265" alt="Screenshot 2024-06-27 at 9 36 32 AM" src="https://github.com/WFP-VAM/prism-app/assets/8203830/853445b0-604a-49b1-a157-d9607d124247">